calculator.py

```diff
@@ -1,39 +1,44 @@
 import sys
 
 try:
-    symbol, x, y = sys.argv[1], sys.argv[2], sys.argv[3]
+    symbol, num1, num2 = sys.argv[1], sys.argv[2], sys.argv[3]
 except:
     print('error')
     sys.exit()
 
-if (not x.isdigit()) or (not y.isdigit()):
+if (not num1.isdigit()) or (not num2.isdigit()):
     print('error')
     sys.exit()
 else:
-    x, y = int(x), int(y)
+    num1, num2 = int(num1), int(num2)
 
 
-def calc(symbol, x, y):
+def calc(symbol, num1, num2):
     if symbol == '+':
-        return print(x + y)
+        result = num1 + num2
     elif symbol == '-':
-        return print(x - y)
+        result = num1 - num2
     elif symbol == '*':
-        return print(x * y)
+        result = num1 * num2
     elif symbol == '/':
-        if y == 0:
-            return print("error")
-        return print(x / y)
+        if num2 == 0:
+            print('error')
+            sys.exit()
```

Comment:
Try not to use `print()` here. As an example, you can return `None` for now. Then print an error message afterward. 

This will separate calculations from the presentation
  -> i.e.  **Single Responsibility Principal**

Please, read this good article 
https://medium.com/@saddamhr/clean-code-functions-should-do-exactly-one-thing-8bf648bc71ea

Reply:
Btw, this principle is a part of five design principals named: **SOLID**. Let's discuss it later
More info:  https://en.wikipedia.org/wiki/SOLID

Reply:
done